src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Program.cs

```diff
@@ -179,12 +179,18 @@ static CliRootCommand CreateCommandLineParser()
         Required = false
     };
 
-    var extensionAssemblyPathsOption = new CliOption<string[]?>("--extension", "--extensions") // TODO: remove plural form
+    var extensionAssemblyPathsOption = new CliOption<string[]?>("--extension")
     {
         Description = "Full paths of extension assemblies to load (optional).",
         Required = false
     };
 
+    var devKitDependencyPathOption = new CliOption<string?>("--devKitDependencyPath")
```

Comment:
Nit, but potentially confusing otherwise: should this be a devKitDependenciesPath? Since there's multiple things in the folder?

Reply:
Maybe?  I chose non-plural because there is a single dependency project.  Not sure